src/web/app/components/question-types/question-edit-details-form/question-edit-details-form.component.ts

```diff
@@ -32,4 +32,11 @@ export abstract class QuestionEditDetailsFormComponent<D extends FeedbackQuestio
   triggerModelChange(field: string, data: any): void {
     this.detailsChange.emit(Object.assign(this.model, { [field]: data }));
```

Comment:
While you are here. I think it is safer to change it to
```suggestion
    this.detailsChange.emit(Object.assign({}, this.model, { [field]: data }));
```